ui/src/services/NodeVolumesUtils.js

```diff
@@ -6,6 +6,7 @@ import {
   STATUS_AVAILABLE,
   STATUS_BOUND,
   STATUS_RELEASED,
+  STATUS_READY,
 } from '../constants';
 
 export const isVolumeDeletable = (rowData, persistentVolumes) => {
```

Comment:
I think you forgot to replace the `STATUS_AVAILABLE` in the outer switch l.22 (for `Volume`).
Note that you should only change this one, the other one in the inner switch l.36 should stay `STATUS_AVAILABLE` (because it's about `PersistentVolume`).